lib/tasks/airtable_data.rake

```diff
@@ -0,0 +1,127 @@
+            expected_attendees modality apac created_at].each do |field|
+            value = record.send field
+            hackathon.send "#{field}=", value
+          end
+
+          # Re-geocoding existing address
+          hackathon.address = record.full_address
+
+          # Attach logo and banner images
+          hackathon.logo.attach(io: record.logo, filename: record.logo_filename) if record.logo
+          hackathon.banner.attach(io: record.banner, filename: record.banner_filename) if record.banner
+
+          hackathon.record(:imported, from: :airtable, data: record.fields.except(airtable_swag_mailing_address_fields))
+
+          # Financial assistance
+          if record.offers_financial_assistance
+            hackathon.tag_with("Offers Financial Assistance")
+          end
+
+          # Swag mailing address
+          if record.swag_mailing_address
+            hackathon.swag_mailing_address_attributes = {
+              **record.swag_mailing_address,
+              created_at: record.created_at
+            }
+
+            original_address = record.fields.slice(airtable_swag_mailing_address_fields)
+
+            hackathon.swag_mailing_address.record(:imported, from: :airtable, data: original_address)
+          end
+
+          # Applicant
+          hackathon.applicant = User.find_or_initialize_by(email_address: record.applicant_email) do |applicant|
+            applicant.created_at = record.created_at
+            applicant.record(:imported, from: :airtable)
+          end
+
+          # Save it!
+          puts "Creating #{hackathon.name} (#{hackathon.airtable_id})"
+          hackathon.save!
+
+          # Flag geocoding errors
+          if hackathon.in_person? && !hackathon.geocoded?
+            hackathon.record(:airtable_migration_geocoding_error, data: {airtable_coordinates: record.coordinates})
+          end
+          if record.coordinates && hackathon.geocoded? && hackathon.distance_to(record.coordinates) > 100
+            hackathon.record(:airtable_migration_geocoding_distance_error, data: {airtable_coordinates: record.coordinates})
+          end
+        end
+      end
+    end
+
+    def migrate_subscriptions
+      puts "MIGRATING SUBSCRIPTIONS"
+      Airtable::Subscriber.all(sort: {created_at: :asc}).each do |record|
+        ActiveRecord::Base.transaction do
+          if (subscription = Hackathon::Subscription.find_by(airtable_id: record.id))
+            puts "Skipping #{record.id} (#{subscription.location})"
+            next
+          end
+
+          subscription = Hackathon::Subscription.new
+          subscription.airtable_id = record.id
+
+          # Re-geocoding existing address
+          subscription.location_input = record.location
+
+          subscription.status = record.status
+          if subscription.inactive? && record.unsubscribed_at
+            subscription.record(:disabled, created_at: record.unsubscribed_at)
```

Comment:
```suggestion
            subscription.record :disabled, unsubscribed_at: record.unsubscribed_at
```

Reply:
the Event's `created_at` should be the time it was unsubscribed so that it shows up correctly on timelines.

This follows the [existing implementation](https://github.com/hackclub/hackathons-backend/blob/1d249824555e29bb14eb50bff3e98a40e5c44bb9/app/models/hackathon/subscription/status.rb#L38-L44) where the `created_at` represents the time it was unsubscribed from.